Formula/gnutls.rb

```diff
@@ -1,14 +1,13 @@
 class Gnutls < Formula
   desc "GNU Transport Layer Security (TLS) Library"
   homepage "https://gnutls.org/"
-  url "https://www.gnupg.org/ftp/gcrypt/gnutls/v3.6/gnutls-3.6.16.tar.xz"
-  mirror "https://www.mirrorservice.org/sites/ftp.gnupg.org/gcrypt/gnutls/v3.6/gnutls-3.6.16.tar.xz"
-  sha256 "1b79b381ac283d8b054368b335c408fedcb9b7144e0c07f531e3537d4328f3b3"
+  url "https://www.gnupg.org/ftp/gcrypt/gnutls/v3.7/gnutls-3.7.2.tar.xz"
+  mirror "https://www.mirrorservice.org/sites/ftp.gnupg.org/gcrypt/gnutls/v3.7/gnutls-3.7.2.tar.xz"
+  sha256 "646e6c5a9a185faa4cea796d378a1ba8e1148dbb197ca6605f95986a25af2752"
   license all_of: ["LGPL-2.1-or-later", "GPL-3.0-only"]
-  revision 1
 
   livecheck do
-    url "https://www.gnupg.org/ftp/gcrypt/gnutls/v3.6/"
+    url "https://www.gnupg.org/ftp/gcrypt/gnutls/v3.7/"
     regex(/href=.*?gnutls[._-]v?(\d+(?:\.\d+)+)\.t/i)
```

Comment:
```suggestion
    url "https://www.gnutls.org/news.html"
    regex(/>\s*GnuTLS\s*v?(\d+(?:\.\d+)+)\s*</i)
```

If we decide to track the "next" version (currently 3.7.x) and want to check the `news.html` page, I would suggest this.